demos/storybook/package.json

```diff
@@ -27,12 +27,12 @@
         "@angular/platform-browser-dynamic": "~12.2.4",
         "@brightlayer-ui/angular-themes": "^6.3.0-beta.0",
         "@brightlayer-ui/colors": "^3.0.1",
+        "@brightlayer-ui/icons": "^1.7.1-beta.1",
```

Comment:
Why are these beta packages?  Since you're in here can you update these to be 1.8.1 ?

Reply:
done. Also upgraded icons-svg